### PR TITLE
MIG-1476: release notes for MTC 1.8.1

### PR DESCRIPTION
**Pull request by anarnold97**
### PR 

* **Release Notes for MTC 1.8.1** - [MIG-1476](https://issues.redhat.com//browse/MIG-1476)                                          

#### Version(s):

* Enterprise 4.11 → branch/enterprise-4.11
* Enterprise 4.12 → branch/enterprise-4.12
* Enterprise 4.13 → branch/enterprise-4.13
* Enterprise 4.14 → branch/enterprise-4.14

#### Issue:

(https://issues.redhat.com/browse/MIG-1476)

### Link to docs preview:

* [MTC 1.8.1 release notes](https://66767--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes#migration-mtc-release-notes-1-8-1_mtc-release-notes)

#### QE review:
- [X](https://github.com/openshift/openshift-docs/pull/66767#pullrequestreview-1693774437) - QE has approved this change.

Additional information:

### CODE REVIEW

**Review by rayfordj — APPROVED** [review]
/lgtm

File: modules/migration-mtc-release-notes-1-8-1.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+= {mtc-full} 1.8.1 release notes
+
+[id="resolved-issues-1-8-1_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+.CVE-2023-39325: golang: net/http, x/net/http2: rapid stream resets can cause excessive work
+
+A flaw was found in handling multiplexed streams in the HTTP/2 protocol, which is utilized by {mtc-full} ({mtc-short}). A client could repeatedly make a request for a new multiplex stream and immediately send an `RST_STREAM` frame to cancel it. This creates additional workload for the server in terms of setting up and dismantling streams, while avoiding any server-side limitations on the maximum number of active streams per connection, resulting in a denial of service due to server resource consumption. link:https://bugzilla.redhat.com/show_bug.cgi?id=2245079[(BZ#2245079)]
```
Comment (tmalove) [inline modules/migration-mtc-release-notes-1-8-1.adoc:15]:
```suggestion
A flaw was found in handling multiplexed streams in the HTTP/2 protocol, which is used by {mtc-full} ({mtc-short}). A client could repeatedly make a request for a new multiplex stream and immediately send an `RST_STREAM` frame to cancel it. This creates additional workload for the server in terms of setting up and dismantling streams, while avoiding any server-side limitations on the maximum number of active streams per connection, resulting in a denial of service due to server resource consumption. link:https://bugzilla.redhat.com/show_bug.cgi?id=2245079[(BZ#2245079)]
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

File: modules/migration-mtc-release-notes-1-8-1.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+It is advised to upgrade to {mtc-short} 1.8.1 or later, which resolve this issue.
```
Comment (skopacz1) [inline modules/migration-mtc-release-notes-1-8-1.adoc:17]: Per our term glossary, [update is preferred over upgrade](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update).
```suggestion
It is advised to update to {mtc-short} 1.8.1 or later, which resolve this issue.
```

Comment (skopacz1) [inline modules/migration-mtc-release-notes-1-8-1.adoc:17]: Small nit:
```suggestion
It is advised to upgrade to {mtc-short} 1.8.1 or later, which resolves this issue.
```

**Review by skopacz1 — COMMENTED** [review]
left a few nits but otherwise looks good to me!

**Comment by jeana-redhat** [comment]
LGTM for merge

**Pull request merged**